pages/common/colon.md

```diff
@@ -0,0 +1,9 @@
+# colon
+
+> Returns a successful exit status code of 0.
+> Use this with the operator `||` to make a command always exit with 0.
```

Comment:
IMO, this does not add much value because whatever is written at the right of operator is not executed unless a redirection fails.

```shell
: || echo "Foo" "Bar" "Baz" # Never executed.
```

Reply:
```suggestion
```